capture/project_library/ot_trace_library/trace_library.py

```diff
@@ -47,11 +47,13 @@ class TraceLibrary:
     """
```

Comment:
I think this change should be factored out into a separate PR.

It would be also good if you could provide some more details on why the database should not be held in memory for multi-processing support. At least to me, it's not obvious why this helps.